src/apps/langchain/pyproject.toml

```diff
@@ -29,8 +29,9 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.9,<3.13"
 trulens-core = { version = "^1.0.0" }
-langchain = ">=0.2.10,!=0.2.12"
-langchain-core = ">=0.2.24,!=0.2.27"
+langchain = "=0.2.11"                 # ,!=0.2.12"
```

Comment:
including 0.2.10?

Reply:
0.2.10 seems to be ok so I relaxed the pin a bit